crates/nargo/src/ops/execute.rs

```diff
@@ -14,6 +15,15 @@ pub fn execute_circuit<B: BlackBoxFunctionSolver>(
 ) -> Result<WitnessMap, NargoError> {
     let mut acvm = ACVM::new(blackbox_solver, circuit.opcodes, initial_witness);
 
+    // Assert messages are not a map due to serde-reflect issues
```

Comment:
I've created an issue for this in acvm that we can link to https://github.com/noir-lang/acvm/issues/522